daml/splice-amulet/daml/Splice/ExternalPartyAmuletRules.daml

```diff
@@ -45,9 +45,8 @@ template ExternalPartyAmuletRules
         nonce : Int
         description : Optional Text
       controller sender
-      do -- TODO(#18633): once the `isLedgerTimeXX` primitives are in, enforce that `expiresAt` is in the future
-         -- We don't do that yet, as otherwise this would be a regression, as it would introduce a time-bound
-         -- on the prepared transaction creating the transfer command.
+      do
+         assertWithinDeadline "ExternalPartyAmuletRules_CreateTransferCommand.expiresAt" expiresAt
```

Comment:
should we have a test for this? it seems like we are enforcing a new property now but none of the tests cover this

Reply:
good point. added.